gr-filter/python/filter/qa_fir_filter.py

```diff
@@ -162,30 +162,42 @@ def test_fir_filter_fsf_001(self):
         taps = 20 * [0.5, 0.5]
         src_data = 40 * [1, 2, 3, 4]
         expected_data = fir_filter(src_data, taps, decim)
-        expected_data = [int(e) for e in expected_data]
+        expected_data = [round(e) for e in expected_data]
 
         src = blocks.vector_source_f(src_data)
         op = filter.fir_filter_fsf(decim, taps)
         dst = blocks.vector_sink_s()
         self.tb.connect(src, op, dst)
         self.tb.run()
         result_data = dst.data()
-        self.assertComplexTuplesAlmostEqual(expected_data, result_data, 5)
+
+        # TODO: Remove fallback once GNU Radio's required VOLK_VERSION is >= 3.1.0
+        try:
+            self.assertEqual(expected_data, result_data)
```

Comment:
I also changed the assertions from `assertComplexTuplesAlmostEqual` to `assertEqual`, since the outputs are integers, not complex numbers.